### PR TITLE
benchmark cyclonedx-bom==5.1.1

### LINKED ISSUE #9: Benchmark:  add CycloneDX-python
add the following tool to the benchmarks: 

github: https://github.com/CycloneDX/cyclonedx-python
pypi: https://pypi.org/project/cyclonedx-bom/
docs: https://cyclonedx-bom-tool.readthedocs.io/
example for 
* venv: `cyclonedx-py environment --of JSON --sv 1.6 -o sbom.cdx.json --PEP-639 --gather-license-texts --pyproject my-pyproject.toml venv`
* requirements file: `cyclonedx-py requirements --of JSON --sv 1.6 -o sbom.cdx.json --pyproject my-pyproject.toml requirements.txt`

### PR DESCRIPTION
**Pull request by jkowalleck**
fixes #9

### CODE REVIEW

File: benchmark/install-all.sh [hunk 1]
```diff
@@ -19,6 +20,8 @@ tar -xzvf "trivy_${TRIVY_VERSION}_Linux-64bit.tar.gz" trivy
 rm "trivy_${TRIVY_VERSION}_Linux-64bit.tar.gz"
 chmod a+x ./trivy
 
+pip install "cyclonedx-bom==${CYCLONEDXBOM_VERSION}"
```
Comment (pombredanne) [inline benchmark/install-all.sh:23]: You may want to install that in another separate virtualenv to avoid an observer effect?

Comment (jkowalleck) [reply benchmark/install-all.sh:23]: you think so? I mean, the system under test is a venv created without `--system-site-packages`, so this should not happen.

From a best-practice perspective, you are right. In generally, decoupling a tool into an own venv is probably the best solution here. 
But there was no existing art for my case of installation, so I'd let this decision/task to @sethmlarson. :-D

Comment (pombredanne) [reply benchmark/install-all.sh:23]: I thought you would be installed in the venv as the code under analysis! ... you are not.

**Review by sethmlarson — APPROVED** [review]
Thank you for contributing this tool! :pray:

**Pull request merged**